### PR TITLE
[INLONG-1771] Website readme error

### PR DESCRIPTION
**Pull request by leezng**
Fixes #apache/incubator-inlong#1771

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by leezng** [comment]
@gosonzhang There is a text missing, please check again.

**Review by gosonzhang — APPROVED** [review]
+1

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ https://inlong.apache.org/
 
 ## Prerequisite
 
-InLong website is powered by [docsite](https://docusaurus.io/).
-If your version of docsite is less than `2.0.0`,please upgrade to `2.0.0`.
-Please also make sure your node version is 14.x, versions higher than 14.x is not supported by docsite yet.
+InLong website is powered by [docusaurus](https://docusaurus.io/).
```
Comment (gosonzhang) [inline README.md:8]: Should the first letter of "docusaurus" be capitalized?
I understand that this modification is to mark the name of our front-end component. If it is all lowercase, I find that it cannot be translated

**Review by gosonzhang — CHANGES_REQUESTED** [review]

**Comment by gosonzhang** [comment]
@leezng, one word feels a problem, please take a look

File: README.md (hunk 1 above)
Comment (leezng) [reply README.md]: done

**Review by gosonzhang — APPROVED** [review]
+1

**Pull request merged**